app/services/payment_providers/adyen_service.rb

```diff
@@ -116,5 +124,15 @@ def reattach_provider_customers(organization_id:, adyen_provider:)
           c.update(payment_provider_id: adyen_provider.id)
         end
     end
+
+    private
+
+    def update_payment_status(event, payment_type)
+      provider_payment_id = event['pspReference']
+      status = event['success'] == 'true' ? 'succeeded' : 'failed'
```

Comment:
Style/TernaryParentheses: Use parentheses for ternary expressions with complex conditions.